README.md

```diff
@@ -172,6 +172,24 @@ Additionally, required for tests:
 5) Required packages:
    - liblevel-zero-dev (Linux) or level-zero-sdk (Windows)
 
+#### DevDax memory provider (Linux only)
+
+A memory provider that provides memory from a device DAX (a character device file /dev/daxX.Y).
+It can be used when gigantic memory mappings are needed.
```

Comment:
nit: `gigantic` seems a little funny(?) in the docs - perhaps just `large`?

Reply:
I took it from https://docs.pmem.io/ndctl-user-guide/ndctl-man-pages/ndctl-create-namespace#theory-of-operation but I can replace it with large if you wish.

Reply:
Done